script/BatchScript.s.sol

```diff
@@ -140,6 +140,9 @@ abstract contract BatchScript is Script {
         } else if (chainId == 2522) {
             SAFE_API_BASE_URL = "";
             SAFE_MULTISEND_ADDRESS = 0x998739BFdAAdde7C933B942a68053933098f9EDa;
+        } else if (chainId == 2810) {
```

Comment:
Looks good. I borrowed this file and modified it, but perhaps I can move these constants to base data to simplify the setup for adding a new chain.
And I think it's worth adding these changes to the readme so it's easier for someone new to add a new network.